Compiler/src/main/java/com/adguard/compiler/Browser.java

```diff
@@ -22,19 +22,14 @@
 	SAFARI("safari"),
 
 	/**
-	 * Firefox browser
+	 * Firefox browser (legacy XPCOM)
 	 */
-	FIREFOX("firefox"),
+	FIREFOX_LEGACY("firefox"),
```

Comment:
Old legacy doesn't exist anymore, also we can call XPCOM extensions as legacy, because it will be deprecated.